packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1116,11 +1137,12 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return !this.disableIsolatedChannels || this.gcEnabled;
     }
 
-    private formMetadata(): IContainerRuntimeMetadata {
+    private formMetadata(): WriteContainerRuntimeMetadata {
         return {
             summaryFormatVersion: 1,
             disableIsolatedChannels: this.disableIsolatedChannels || undefined,
-            gcFeature: this.summaryGCVersion, // retain value, this is unchangeable for nown
+            gcFeature: this.summaryGCVersion, // retain value, this is unchangeable for now
+            sequenceNumber: this.deltaManager.lastSequenceNumber,
```

Comment:
Is it guaranteed that this will match the `referenceSequenceNumber` in the summary? We do pause the delta queues when summarizing but we should assert when writing this in the summary that this matches the `referenceSequenceNumber`.

Reply:
I was thinking about this too.

We could parameterize `formMetadataBlob(sequenceNumber: number)` and pass in `this.deltaManager.lastSequenceNumber` from the summarize call. This is functionally equivalent, and it _might_ guard for future changes, but we already are **heavily** assuming that DeltaManager remains paused throughout the summarize process. We already assert between each async step in the outer `submitSummary` call. So that's why I opted to leave it as-is. What do you think? I am sort of against adding a new assert, but I'd be open to parameterizing it.

Reply:
If we are already validating this, then its fine; we don't need to add another assert.